OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -50,6 +50,9 @@ public class ProductionQueueInfo : ITraitInfo
 		[Desc("The build time is multiplied with this percentage on low power.")]
 		public readonly int LowPowerModifier = 100;
 
+		[Desc("Productions will be build infinitely till canceling after putting more than this amount in the queue.")]
```

Comment:
The description here doesn't make sense.
I suggest `"Production items that have more than this many items in the queue will be produced in a loop"`.